discovery_engine_core/core/src/stack/ops/trusted.rs

```diff
@@ -114,8 +112,7 @@ impl Ops for TrustedNews {
 
     fn merge(&self, stack: &[Document], new: &[Document]) -> Result<Vec<Document>, GenericError> {
         let merged = chain!(stack, new).cloned().collect();
-        let filtered = filter_semantically(merged, &self.semantic_filter_config);
-        Ok(filtered)
+        Ok(merged)
```

Comment:
:art: similarly this could be written as a one liner now
```suggestion
        Ok(chain!(stack, new).cloned().collect())
```